### ISSUE TITLE
Building latex docs fails in the release script

### ISSUE THREAD

**Issue by oscarbenjamin** [sympy/sympy#20307]
The release script failed with:
```
The LaTeX files are in _build/latex.
Run 'make' in that directory to run these through (pdf)latex
(use `make latexpdf' here to do that automatically).

Build finished; the LaTeX files are in _build/latex.
Set the environment variable LATEXMKOPTS='-xelatex -silent'
And run `make all' in that directory to run these through xelatex.
latexmk -pdf -dvi- -ps-  'sympy-1.7.rc1.tex'
Latexmk: This is Latexmk, John Collins, 1 January 2015, version: 4.41.
Latexmk: applying rule 'pdflatex'...
Rule 'pdflatex': Rules & subrules not known to be previously run:
   pdflatex
Rule 'pdflatex': The following rules & subrules became out-of-date:
      'pdflatex'
------------
Run number 1 of rule 'pdflatex'
------------
------------
Running 'pdflatex   -recorder  "sympy-1.7.rc1.tex"'
------------
This is pdfTeX, Version 3.14159265-2.6-1.40.17 (TeX Live 2016/Debian) (preloaded format=pdflatex)
 restricted \write18 enabled.
entering extended mode
(./sympy-1.7.rc1.tex
LaTeX2e <2017/01/01> patch level 3
Babel <3.9r> and hyphenation patterns for 3 language(s) loaded.
(./sphinxmanual.cls
Document Class: sphinxmanual 2019/12/01 v2.3.0 Document class (Sphinx manual)
(/usr/share/texlive/texmf-dist/tex/latex/base/report.cls
Document Class: report 2014/09/29 v1.4h Standard LaTeX document class
(/usr/share/texlive/texmf-dist/tex/latex/base/size10.clo)))
(/usr/share/texlive/texmf-dist/tex/latex/cmap/cmap.sty)
(/usr/share/texlive/texmf-dist/tex/latex/tools/bm.sty)
(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amssymb.sty
(/usr/share/texlive/texmf-dist/tex/latex/amsfonts/amsfonts.sty))
(/usr/share/texlive/texmf-dist/tex/latex/fontspec/fontspec.sty
(/usr/share/texlive/texmf-dist/tex/latex/l3kernel/expl3.sty
(/usr/share/texlive/texmf-dist/tex/latex/l3kernel/expl3-code.tex)
(/usr/share/texlive/texmf-dist/tex/latex/l3kernel/l3pdfmode.def))
(/usr/share/texlive/texmf-dist/tex/latex/l3packages/xparse/xparse.sty)

!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
!
! Fatal fontspec error: "cannot-use-pdftex"
! 
! The fontspec package requires either XeTeX or LuaTeX.
! 
! You must change your typesetting engine to, e.g., "xelatex" or
! "lualatex"instead of plain "latex" or "pdflatex".
! 
! See the fontspec documentation for further information.
! 
! For immediate help type H <return>.
!...............................................  
                                                  
l.28 \msg_fatal:nn {fontspec} {cannot-use-pdftex}
                                                 
?
```
Testing locally I find that make fails but following the command suggested in the error messages I get:
```
$ export LATEXMKOPTS='-xelatex -silent'
$ make all
latexmk -pdf -dvi- -ps- -xelatex -silent 'sympy-1.7.rc1.tex'
Latexmk: Run number 1 of rule 'xelatex'
This is XeTeX, Version 3.14159265-2.6-0.99998 (TeX Live 2017) (preloaded format=xelatex)
 restricted \write18 enabled.
entering extended mode
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif/BI', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif/B', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif/I', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans/BI', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans/B', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans/I', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono/BI', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono/B', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono/I', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Serif', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
kpathsea:make_tex: Invalid fontname `DejaVu Sans Mono', contains ' '
Latexmk: Summary of warnings:
  Latex failed to resolve 25 reference(s)
Collected error summary (may duplicate other messages):
  xelatex: Command for 'xelatex' gave return code 1
      Refer to 'sympy-1.7.rc1.log' for details
Latexmk: Use the -f option to force complete processing,
 unless error was exceeding maximum runs of latex/pdflatex.
make: *** [sympy-1.7.rc1.pdf] Error 12
```

**Comment by oscarbenjamin** [comment]
It seems that the latex build is broken generally not just in the release script (unless I'm just not running it in the right way).

**Comment by oscarbenjamin** [comment]
I've merged #20309 which possibly fixes this issue on the 1.7 branch. I will test the release script now. If the fix works then this should also be cherry-picked back to master.

**Comment by oscarbenjamin** [comment]
This issue can be closed when #20457 is merged.

**Comment by oscarbenjamin** [comment]
Fixed by #20443
